src/components/Icon/Icon.tsx

```diff
@@ -879,15 +901,143 @@ const Icon = ({
         </>
       ) : null}
 
+      {name === 'critical-solid' ? (
+        <>
+          <path
+            fill={color}
+            stroke="none"
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M24 12C24 18.6274 18.6274 24 12 24C5.37258 24 0 18.6274 0 12C0 5.37258 5.37258 0 12 0C18.6274 0 24 5.37258 24 12ZM12 6C12.5523 6 13 6.44772 13 7V14C13 14.5523 12.5523 15 12 15C11.4477 15 11 14.5523 11 14V7C11 6.44772 11.4477 6 12 6ZM12 19C12.6904 19 13.25 18.4404 13.25 17.75C13.25 17.0596 12.6904 16.5 12 16.5C11.3096 16.5 10.75 17.0596 10.75 17.75C10.75 18.4404 11.3096 19 12 19Z"
+          />
+        </>
+      ) : null}
+
+      {name === 'critical' ? (
+        <>
+          <line
+            x1="12"
+            y1="7"
+            x2="12"
+            y2="14"
+            stroke-width="2"
+            stroke-linecap="round"
+          />
+          <circle cx="12" cy="17.75" r="1.25" fill="black" />
+          <circle cx="12" cy="12" r="11" stroke="black" stroke-width="2" />
+        </>
+      ) : null}
+
+      {name === 'fx-account' ? (
+        <>
+          <path
+            fill={color}
+            stroke="none"
+            d="M23.5424 8.71316V8.70941C23.5306 8.66869 23.5183 8.62797 23.506 8.58725C23.5033 8.57707 23.5001 8.56689 23.4969 8.55724C23.4738 8.48009 23.4503 8.40293 23.4256 8.32631L23.4224 8.3156C23.2721 7.84854 23.0931 7.39118 22.8866 6.9461C22.6865 6.50528 22.4556 6.07915 22.1954 5.6709C21.1906 4.05672 19.8184 2.70293 18.1909 1.71993V1.71993C16.3226 0.591513 14.1808 -0.00333171 11.9981 1.40362e-05C11.6231 1.40362e-05 11.248 0.0182312 10.8826 0.0535938C9.59025 0.189687 7.41169 0.654223 5.6441 2.20643C12.0785 -1.07908 20.6748 2.87725 19.9435 10.2338C19.6831 12.8592 17.4542 14.6632 15.379 14.9675C15.0099 15.0239 14.6347 15.0277 14.2645 14.9788C9.19643 14.2742 9.50666 10.1202 11.0465 8.34399C9.01051 8.33114 7.43152 10.851 8.19878 13.3269C6.43333 10.0489 8.21753 4.91489 11.9992 3.36107C7.55475 2.71812 3.38839 4.87202 2.04943 8.61565C2.20933 7.24725 2.5285 5.90222 3.00047 4.60788C3.00047 4.60788 0.57759 6.52282 0.0835846 10.5499C0.0760834 10.612 0.069118 10.6742 0.0626885 10.7369C0.0600095 10.7583 0.0578662 10.7797 0.055723 10.8017C0.0367915 10.9892 0.0225036 11.178 0.0128592 11.368C0.0128592 11.3825 0.0128592 11.397 0.0128592 11.4114C0.00910866 11.4907 0.00589376 11.5722 0.00375057 11.6504V11.7174C0.00375057 11.8117 0 11.906 0 12.0008C0 18.6292 5.37352 24.0027 12.0019 24.0027C18.6302 24.0027 24 18.6302 24 12.0019C24.0006 10.8897 23.8466 9.78291 23.5424 8.71316V8.71316Z"
+          />
+        </>
+      ) : null}
```

Comment:
Is there a plan to DRY this at some point? Seems like a lot of copy-pasting already. I suppose ideally the svg content would live in separate files and be imported in here, instead of inlining all of them. Would be nice if this component was mainly just a list of icon names.

Reply:
You could even go as far as generating a json file from a directory listing of SVG files, and then use that as imported data to render icons in a generalized way. You could then use the same json data to display all the icons in Storybook for verification.
Just thoughts for improvements. No need to actually change anything in this PR.

Reply:
yea this is a good one to think about. I'm considering making all the icons their own components to abstract a lot of this code, Rollup  is a bit odd for imports and how it creates the lib.. still trying to figure out the best way. Idealy id use an svg sprite but doesnt seem like a possibility with Rollup.